### PR TITLE
Selenium Chrome Billy Driver

### PR DESCRIPTION
**Pull request by eliotsykes**

### CODE REVIEW

**Comment by eliotsykes** [comment]
I've used this successfully without `--ignore-certificate-errors`, although perhaps it'll be needed at a later date by others. http://peter.sh/experiments/chromium-command-line-switches/#ignore-certificate-errors

**Comment by eliotsykes** [comment]
This doesn't seem to be working as intended yet. Not ready, apologies!

**Comment by ronwsmith** [comment]
Off to a good start though! Let me know when it's ready to go.

**Comment by eliotsykes** [comment]
Thanks @ronwsmith :+1: - this commit seems to have done it. I'd got the --proxy-server option syntax wrong before oops!

File: lib/billy.rb [hunk 1]
```diff
@@ -59,6 +59,13 @@ def self.register_drivers
           ssl: "#{Billy.proxy.host}:#{Billy.proxy.port}")
         Capybara::Selenium::Driver.new(app, profile: profile)
       end
+
+      Capybara.register_driver :selenium_chrome_billy do |app|
+        Capybara::Selenium::Driver.new(
+          app, :browser => :chrome,
```
Comment (ronwsmith) [inline lib/billy.rb]: Looks good!  Can we remove the hash rockets on these two lines?

Comment (eliotsykes) [inline lib/billy.rb]: Oops again! Updated and thanks @ronwsmith

**Comment by ronwsmith** [comment]
v0.5.1 is yours!

**Comment by eliotsykes** [comment]
Fantastic, cheers! Noted to use the puffing-billy 0.5.1 as a result of this, thank you @ronwsmith https://github.com/eliotsykes/rspec-rails-examples/issues/52

**Pull request merged**